src/libraries/System.Text.Json/Common/JsonKnownNamingPolicy.cs

```diff
@@ -21,6 +21,26 @@ enum JsonKnownNamingPolicy
         /// <summary>
         /// Specifies that the built-in <see cref="Json.JsonNamingPolicy.CamelCase"/> be used to convert JSON property names.
         /// </summary>
-        CamelCase = 1
+        CamelCase = 1,
+
+        /// <summary>
+        /// Specifies that the built-in <see cref="Json.JsonNamingPolicy.SnakeLowerCase"/> be used to convert JSON property names.
+        /// </summary>
+        SnakeLowerCase = 2,
```

Comment:
I know you're aware of this, but adding comment to track renamed APIs

```suggestion
        SnakeCaseLower = 2,
```